README.md

```diff
@@ -13,6 +13,19 @@ npm install
 npm start
 ```
 
+## Tasks
+
+- `npm start` Run web server.
```

Comment:
Same as the `watch` one we are now running three tasks for this task which is running the web server, compile LESS and JS then start watch